src/js/components/AppListItemComponent.jsx

```diff
@@ -363,7 +363,7 @@ var AppListItemComponent = React.createClass({
         model.labels["SERVICE_SCHEME"] === "https"))
       ? model.labels["SERVICE_SCHEME"]
       : "http";
-    var cleanAppName = model.id.substring(1).replace("/", "-");
+    var cleanAppName = model.id.substring(1).replace(/\//g, "-");
```

Comment:
FAR: make the service links configurable using the runtimeConfig.js like logs links.